colonyzer2/detectlid.py

```diff
@@ -53,34 +53,34 @@ def checkMatches(fname,posfeats,negfeats,frects,orb,pmatchdist=20,nmatchdist=20,
     negmatches=getMatches(fname,negfeats,frects,orb,draw)
     testmatches=testMatches(posmatches,negmatches,pmatchdist,nmatchdist)
     if draw:
-        for i,gm,feat in zip(range(0,len(posfeats)),posmatches,posfeats):
+        for i,gm,feat in zip(list(range(0,len(posfeats))),posmatches,posfeats):
```

Comment:
Not necessary to convert iterator to list for zip?  This should work?

for i,gm,feat in zip(range(0,len(posfeats)),posmatches,posfeats):